doc/source/building/working_with_images/uninstall_package_requests.rst

```diff
@@ -0,0 +1,110 @@
+.. _gracefully_uninstall:
+
+Gracefully Uninstall System Packages
+====================================
+
+.. sidebar:: Abstract
+
+   This page provides some details about *uninstall*
+   package requests and how it could be used in order to remove
+   packages once the image configuration using the `config.sh`
+   script is done.
+
+Uninstalling packages from the system image that were previously installed
+during the installation phase is an operation that can be
+handy under certain circumstances. As an example, someone could be interested
+in performing some configuration tasks in the
+`config.sh` script (see :ref:`prepare step <prepare-step>` for
+further details) that require the inclusion of some extra packages,
+which will be only needed at build time. One example would be compiling
+some unpacked application sources.
+
+KIWI description file schema defines package requests of type `uninstall`
+and type `delete` in order to remove previously installed packages. The
+`uninstall` requests perform a clean packages removal by removing any
+package dependent on the requested ones and also removing orphan
+dependencies. The `delete` requests perform a hard removal without any
```

Comment:
Are there more than one delete requests? I would use the singular form: "The delete request performs ..."